tests/openedx_tagging/core/tagging/test_views.py

```diff
@@ -590,7 +590,7 @@ def test_object_tags_remaining_http_methods(
             response = self.client.post(url, {"test": "payload"}, format="json")
         elif http_method == "PATCH":
             response = self.client.patch(url, {"test": "payload"}, format="json")
-        elif http_method == "DELETE":
+        else:  # http_method == "DELETE"
```

Comment:
This is the only change I'm not sure about... how do we know it's `DELETE` here and not `PUT` for example? Especially for something as important as deleting, I'd prefer to be explicit about matching the DELETE method before assume it's delete.

Reply:
This should make it better: [868057d](https://github.com/openedx/openedx-learning/pull/90/commits/868057dcba4471f89e62fe7afa4bb9798175d967)